web/src/LogPane.scss

```diff
@@ -1,16 +1,27 @@
 @import "constants.scss";
 
 .LogPane {
-  margin: $spacing-unit / 2;
+  position: fixed;
```

Comment:
hmmm...i understand that using "position: fixed" is a cheap fix to force the layout to do what you want.

But in practice, putting the main content of a page inside a position:fixed container is almost always something you'll regret. It's semantically / accessibility-wise not what we want, and going to break the interaction in lots of subtle and frustrating ways.

Let's talk about this more tomorrow.